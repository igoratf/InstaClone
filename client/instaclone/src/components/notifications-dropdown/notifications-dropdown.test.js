import React from 'react';
import { shallow } from 'enzyme';
import NotificationsDropdown from './notifications-dropdown.component';

describe('Notifications dropdown component', () => {
  it('should render notifications dropdown', () => {
    expect(shallow(<NotificationsDropdown />)).toMatchSnapshot();
  })
})