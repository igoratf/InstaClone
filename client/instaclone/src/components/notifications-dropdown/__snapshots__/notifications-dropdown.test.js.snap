// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notifications dropdown component should render notifications dropdown 1`] = `ShallowWrapper {}`;
